windows_containers/windows-containers-release-notes.adoc

```diff
@@ -1,17 +1,22 @@
 [id="windows-containers-release-notes"]
-= Red Hat OpenShift for Windows Containers release notes
+= Windows Container Support for Red Hat OpenShift release notes
 include::modules/common-attributes.adoc[]
 :context: windows-containers-release-notes
 
 toc::[]
 
 [id="about-windows-containers"]
-== About Red Hat OpenShift for Windows Containers
+== About Windows Container Support for Red Hat OpenShift
 
-Red Hat OpenShift for Windows Containers is a feature providing the ability to run Windows compute nodes in an {product-title} cluster. This is possible by using the Red Hat Windows Machine Config Operator (WMCO) to install and manage Windows nodes. With Windows nodes available, you can run Windows container workloads in {product-title}.
+Windows Container Support for Red Hat OpenShift is a feature providing the ability to run Windows compute nodes in an {product-title} cluster. This is possible by using the Red Hat Windows Machine Config Operator (WMCO) to install and manage Windows nodes. With Windows nodes available, you can run Windows container workloads in {product-title}.
 
 The release notes for Red Hat OpenShift for Windows Containers tracks the development of the WMCO, which provides all Windows container workload capabilities in {product-title}.
 
+[id="getting-support"]
+== Getting support
+
+You must have a subscription to receive support for the Red Hat WMCO. Deploying Windows container workloads in production clusters is not supported without a subscription. If you do not have a subscription, you can use the community WMCO, a distribution that lacks official support. Support should be requested through the link:http://access.redhat.com/[Red Hat Customer Portal].
```

Comment:
> ! should v
```suggestion
You must have a subscription to receive support for the Red Hat WMCO. Deploying Windows container workloads in production clusters is not supported without a subscription. If you do not have a subscription, you can use the community WMCO, a distribution that lacks official support. Request support through the link:http://access.redhat.com/[Red Hat Customer Portal].
```
Always prefer a more direct verb or way of saying something. Omit “should,” or use “must” or “might,” as appropriate.